src/test/java/com/wikia/webdriver/common/core/urlbuilder/UrlBuilder.java

```diff
@@ -64,22 +74,33 @@ private String getUrlPrefix(String wikiName) {
   }
 
   private String getUrlSuffix(String wikiName) {
+    if (env.contains("dev") && isMercuryBrowser()) {
+      return String.format(XIPIO_ADDRESS_FORMAT, XIPIO_DEFAULT_DOMAIN, XIPIO_DEFAULT_PORT);
```

Comment:
The readability reason depends on reader, I guess ;) Since IMHO the names of constants are self-explanatory. The idea was to make it possible in future to pass custom domain and port.

We're using this consts only once, so it might be interpreted as over-engineering :) Although, I found using constants a good practice in Java. I'll leave it for now since we want to merge it ASAP and the discussion about constants vs explicit strings can go on and on and on :japanese_ogre: